routes/userRoutes.js

```diff
@@ -12,6 +19,9 @@ const router = express.Router();
 router.route('/user')
   .all(verifyToken)
   .get(getUser)
-  .put(updateUser);
+  .patch(passwordValidator, isUsernameValidator, usernameExistsValidator,
```

Comment:
step these middlewares to new lines like the ones on lines 23, 24, 25

Reply:
ok